puppet/modules/slave/manifests/init.pp

```diff
@@ -193,33 +193,33 @@
 
   # Databases
   include slave::mysql, slave::postgresql
-  slave::db_config { "mysql": }
-  slave::db_config { "sqlite3": }
-  slave::db_config { "postgresql": }
+  slave::db_config { 'mysql': }
+  slave::db_config { 'sqlite3': }
+  slave::db_config { 'postgresql': }
 
   # RVM
   class { 'rvm':
     version => '1.26.11',
   }
   if $rvm_installed == "true" {
-    rvm::system_user { "jenkins": }
-    slave::rvm_config { "ruby-1.8.7":
-      version => "ruby-1.8.7-p371",
+    rvm::system_user { 'jenkins': }
+    slave::rvm_config { 'ruby-1.8.7':
+      version => 'ruby-1.8.7-p371',
     }
-    slave::rvm_config { "ruby-1.9.2":
-      version => "ruby-1.9.2-p320",
+    slave::rvm_config { 'ruby-1.9.2':
+      version => 'ruby-1.9.2-p320',
     }
-    slave::rvm_config { "ruby-1.9.3":
-      version => "ruby-1.9.3-p392",
+    slave::rvm_config { 'ruby-1.9.3':
+      version => 'ruby-1.9.3-p392',
     }
-    slave::rvm_config { "ruby-2.0.0":
-      version => "ruby-2.0.0-p353",
+    slave::rvm_config { 'ruby-2.0.0':
+      version => 'ruby-2.0.0-p353',
     }
-    slave::rvm_config { "ruby-2.1":
-      version => "ruby-2.1.2",
+    slave::rvm_config { 'ruby-2.1':
+      version => 'ruby-2.1.5',
     }
-    slave::rvm_config { "ruby-2.2":
-      version => "ruby-2.2.1",
+    slave::rvm_config { 'ruby-2.2':
+      version => 'ruby-2.2.2',
```

Comment:
We're on 1.26.11 (the latest) at the moment, but it doesn't have 2.2.2 available.  (I suspect this might help https://github.com/theforeman/foreman/pull/2517 too, but we're a bit stuck.)